pontoon/settings/base.py

```diff
@@ -164,25 +170,25 @@ def path(*args):
                 'django_jinja.builtins.extensions.DjangoFiltersExtension',
                 'pipeline.templatetags.ext.PipelineExtension',
             ],
-            'globals': {
-                'browserid_info': 'django_browserid.helpers.browserid_info',
-            }
         }
     },
     {
         'BACKEND': 'django.template.backends.django.DjangoTemplates',
-        'DIRS': [],
-        'APP_DIRS': True,
+        'DIRS': [path('pontoon/templates')],
```

Comment:
You're totally right -> django-allauth currently don't provide any support for jinja2 tags and I couldn't figure out how to make django-templates use our jinja2 templates.